fdbclient/include/fdbclient/BulkLoading.h

```diff
@@ -926,4 +930,36 @@ BulkLoadJobState createBulkLoadJob(const UID& dumpJobIdToLoad,
                                    const std::string& jobRoot,
                                    const BulkLoadTransportMethod& transportMethod);
 
+// When the storage engine is not ShardedRocksDB, SS conducts bulkLoad using fetchKey based method.
+// In this case, SS needs to persist the bulkload task metadata locally because when SS restarts, SS
+// does not have the bulkload task information. So, we need to persist this information to SS disk locally.
+// This data structure is the metadata persisted at SS special key space locally.
```

Comment:
There is a local special key space per SS?

Reply:
Yes. Search `PERSIST_PREFIX` in storage servers.